src/Bicep.Core/Emit/ResourceDependency.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Bicep.Core.Semantics;
+using Bicep.Core.Syntax;
+
+namespace Bicep.Core.Emit
+{
+    public class ResourceDependency
```

Comment:
Is this specific to resources or modules as well?

Reply:
The thing that uses it is called `ResourceDependencyVisitor` and it applies to modules too and I didn't add that one :P

But seriously though, I've been needing a name for "resource or module" and haven't come up with a good one-word one yet. 

---
In reply to: [582210356](https://github.com/Azure/bicep/pull/1521#discussion_r582210356) [](ancestors = 582210356)